File: import_games.R
txtFile<-readLines('import_games.txt')
for (i in 1:length(txtFile)) {
  gameHash<-substr(txtFile[i],68,103) #68,103 for NA/EU, 68,99 for KR, 67/98 for TW
  idNum<-substr(gameHash,1,10) #10 for NA/EU, 6 for KR
  fullURL<-paste("https://acs.leagueoflegends.com/v1/stats/game/TRLH1/",gameHash,sep="")
  timeHash<-gsub('\\?','/timeline\\?',gameHash)
  timeURL<-paste("https://acs.leagueoflegends.com/v1/stats/game/TRLH1/",timeHash,sep="")
  
  download.file(fullURL,paste("GameData/2016/NA/one/",idNum,sep=""))
  download.file(timeURL,paste("GameData/2016/NA/one/timeline/",idNum,sep=""))
}

File: tools/import_games.m
% Import list of games

gameURL = {
    'http://matchhistory.na.leagueoflegends.com/en/#match-details/TRLH1/1001560015?gameHash=314d26a7b9fb3223&tab=overview';
    'http://matchhistory.na.leagueoflegends.com/en/#match-details/TRLH1/1001560025?gameHash=229b7f6920b359b2&tab=overview';
    'http://matchhistory.na.leagueoflegends.com/en/#match-details/TRLH1/1001560037?gameHash=d4d1fd94568047ab&tab=overview'
    };

for i=1:length(gameURL)
    gameHash = gameURL{i}(62:103);
    idNumber = gameHash(7:16);
    fullURL = strcat('https://acs.leagueoflegends.com/v1/stats/game/',gameHash);
    timeHash = strrep(gameHash,'?','/timeline?');
    timeURL = strcat('https://acs.leagueoflegends.com/v1/stats/game/',timeHash);
    
    websave(strcat('C:\Users\Danny\Documents\onenexus\GameData\NA\Four\',idNumber),fullURL);
    websave(strcat('C:\Users\Danny\Documents\onenexus\GameData\NA\Four\timeline\',idNumber),timeURL);
end